interactive-calculator/src/containers/App/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import BugetSet from '../BugetSet/BudgetSet';
+import ItemsForm from '../ItemsForm/ItemsForm';
+import './App.scss';
+
+const App = () => {
+
+  const [budget, setBudget] = useState(100.0);
+  const [split, setSplit] = useState(1);
+
+  const [items, setItems] = useState([{
```

Comment:
The use of default state here results in the first input returning these parameters. I would recommend setting them to null at first

Reply:
Piggybacking off that you could set the placeholder of the BudgetSet component to be these default values and get the same effect